src/reducers/api.tsx

```diff
@@ -9,7 +9,7 @@ type SetAuthSessionIdPayload = {
 };
 
 export const actions = {
-  setAuthSessionId: createAction('SET_AUTH_TOKEN', (resolve) => {
+  setAuthSessionId: createAction('SET_AUTH_SESSION_ID', (resolve) => {
```

Comment:
I found this with a search but tbh I'm not 100% what is it for - we don't reference it elsewhere in the code.

Reply:
I assume we're okay to change it then

Reply:
It's just an internal constant to identify the action. It makes sense that it isn't used anywhere else.